### ISSUE TITLE
signed as IDP generic

### ISSUE THREAD

**Issue by ghost** [gurnec/HashCheck#34]
Hi,

during the installation 
my trustworthy Avast warned me by a pop up saying
C:\Users<username>\AppData\Local\Temp\NSU14F.TMP => IDP.Generic

![avastui_2017-08-10_21-39-46](https://user-images.githubusercontent.com/30388530/29191860-06d80570-7e17-11e7-93c9-e2c15f1207c7.png)

**Comment by nastyagrifon** [comment]
That's false positive. Avast marks it "IDP.Generic" for suspicious behavior just like AVG Antivirus does.

You can ignore it or whitelist the application. Sometimes updating antivirus software helps as well
